quotesify.log

```diff
@@ -42,3 +42,67 @@ Oct 13, 2020 11:02:04 PM seedu.duke.commands.AddCommand addCategoryToBook
+��Ϣ: going to add book to booklist
+10�� 14, 2020 5:25:33 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 5:47:26 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 5:47:26 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:11:40 ���� seedu.duke.commands.AddCommand addToDo
+��Ϣ: add task to toDoList failed
+10�� 14, 2020 6:12:03 ���� seedu.duke.commands.AddCommand addToDo
+��Ϣ: add task to toDoList failed
+10�� 14, 2020 6:12:26 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:12:26 ���� seedu.duke.commands.AddCommand addBook
+��Ϣ: add book to booklist failed
+10�� 14, 2020 6:12:26 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:12:40 ���� seedu.duke.commands.AddCommand addToDo
+��Ϣ: add task to toDoList failed
+10�� 14, 2020 6:21:05 ���� seedu.duke.commands.AddCommand addToDo
+��Ϣ: add task to toDoList failed
+10�� 14, 2020 6:39:51 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:39:51 ���� seedu.duke.commands.AddCommand addBook
+��Ϣ: add book to booklist failed
+10�� 14, 2020 6:39:51 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:40:01 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:40:01 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:49:11 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:49:11 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:51:24 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:51:24 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:51:31 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:51:31 ���� seedu.duke.commands.AddCommand addBook
+��Ϣ: add book to booklist failed
+10�� 14, 2020 6:51:31 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:51:44 ���� seedu.duke.commands.BookmarkCommand handleBookmark
+��Ϣ: add bookmark to bookmarkList failed
+10�� 14, 2020 6:53:41 ���� seedu.duke.commands.BookmarkCommand handleBookmark
+��Ϣ: add bookmark to bookmarkList failed
+10�� 14, 2020 6:54:53 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:54:53 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
+10�� 14, 2020 6:55:37 ���� seedu.duke.commands.BookmarkCommand handleBookmark
+��Ϣ: add bookmark to bookmarkList failed
+10�� 14, 2020 6:57:52 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: going to add book to booklist
+10�� 14, 2020 6:57:52 ���� seedu.duke.commands.AddCommand execute
+��Ϣ: added book to booklist
```

Comment:
I think the question mark is caused by my default language of the system (CHINESE), i will look for ways to fix that